tests/Feature/Events/CreateEventTest.php

```diff
@@ -711,4 +711,30 @@ public function notifications_are_queued_as_expected()
         # Should have queued ApproveEvent.
         self::assertEquals(0, Queue::size('database'));
     }
+
+    /** @test */
+    public function network_coordinator_other_group() {
+        $network = Network::factory()->create();
+
+        // Create a group in the network.
+        $groupInNetwork = Group::factory()->create();
+        $network->addGroup($groupInNetwork);
+
+        $coordinator = User::factory()->networkCoordinator()->create();
+        $network->addCoordinator($coordinator);
+
+        $this->actingAs($coordinator);
+
+        // Create a group not in the network.
+        $idgroup = $this->createGroup();
+        $groupNotInNetwork = Group::findOrFail($idgroup);
```

Comment:
The code above in User.php looks correct, but this test doesn't look quite right to me - shouldn't the user be made a host of $groupNotInNetwork?

So in theory this test should currently be failing, because this user shouldn't be seeing $groupNotInNetwork as it stands, given it's not in their network and they're not a host of it?

Reply:
Oh, does $this->createGroup() create the group on behalf of the current user?

(I looked, and yes, yes it does.  So all good!)